source/examples/xgboost-dask-databricks/notebook.ipynb

```diff
@@ -486,7 +480,10 @@
     "# Check if the file already exists\n",
     "if not os.path.exists(file_path):\n",
     "    # If not, download dataset to the directory\n",
-    "    download_command = f\"curl https://archive.ics.uci.edu/ml/machine-learning-databases/00280/HIGGS.csv.gz --output {file_path}\"\n",
+    "    data_url = (\n",
+    "        \"https://archive.ics.uci.edu/ml/machine-learning-databases/00280/HIGGS.csv.gz\"\n",
+    "    )\n",
```

Comment:
I made the change to pull this URL out into a separate variable (to fix the line-length warning), but `black-jupyter` chose to add these parentheses.